docker/dockerfile-build-centos

```diff
@@ -2,14 +2,15 @@
 ARG base_image
 
 FROM ${base_image}
-LABEL maintainer="Nico Trost"
+LABEL maintainer="rochpcg-maintainer@amd.com"
```

Comment:
@saadrahim is this the correct maintainer

Reply:
yes, but you will need to make a distribution list for it, it doesn't exist now.